javalib/src/main/scala/java/util/ArrayDeque.scala

```diff
@@ -0,0 +1,241 @@
+/*
+ * Scala.js (https://www.scala-js.org/)
+ *
+ * Copyright EPFL.
+ *
+ * Licensed under Apache License 2.0
+ * (https://www.apache.org/licenses/LICENSE-2.0).
+ *
+ * See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.
+ */
```

Comment:
You can remove that header and just have a comment `// Ported from Scala.js`, like all the other files that have been ported from Scala.js.

Reply:
Done.